products/ghc.md

```diff
@@ -0,0 +1,122 @@
+---
+title: GHC
```

Comment:
Can we use the full title here? I understand GHC is the standard name, but I really want to make sure `haskell` is included in the title.

```suggestion
title: Glasgow Haskell Compiler (GHC)
```